### PR TITLE
add all files from EDDE repo

### PR DESCRIPTION
**Pull request by damonmcc**
related to #520 

this copies nearly all files from the EDDE repo [`db-equitable-development-tool`](https://github.com/NYCPlanning/db-equitable-development-tool). I left out jupyter notebooks and R scripts

will use and revise files in follow-up PRs

### CODE REVIEW

**Comment by fvankrieken** [comment]
Let's maybe take the tests out for now

File: products/edde/notebooks/2012_5YearEstimates_PUMA_Column.ipynb
Comment (fvankrieken) [inline products/edde/notebooks/2012_5YearEstimates_PUMA_Column.ipynb:1]: 🙅

**Comment by fvankrieken** [comment]
My gut is that "internal review" and "external review" folders should not be included, they seem to be output data only?

File: products/edde/notebooks/2012_5YearEstimates_PUMA_Column.ipynb
Comment (damonmcc) [reply products/edde/notebooks/2012_5YearEstimates_PUMA_Column.ipynb:1]: oops

**Comment by damonmcc** [comment]
> My gut is that "internal review" and "external review" folders should not be included, they seem to be output data only?

`external_review/export_DO.sh` is used to export data in the two github actions

and I'm seeing `from internal_review. ... import ...` show up in 34 files in `aggregate/`

**Comment by damonmcc** [comment]
our python formatting test fails, will try formatting those files

**Comment by fvankrieken** [comment]
> @fvankrieken should be good to go
> 
> would rather keep the tests. they aren't being run yet and I'll see how broken they are in a follow-up

Oh yeah definitely keep them, my language was a bit too harsh. Just don't run them in our main test action is all I meant

**Review by fvankrieken — APPROVED** [review]

**Pull request merged**